search/apps.py

```diff
@@ -0,0 +1,17 @@
+"""
+AppConfig
+"""
+from django.apps import AppConfig
```

Comment:
let's be consistent about whether or not we're inserting an empty line before the imports